interfaces/api/v1/core/export.py

```
@@ -98,3 +100,41 @@ async def export_chapter(
+    Returns:
+        流式响应，包含导出的文件
+    """
+    try:
+        # 验证格式
+        if format not in valid_library_formats:
+            raise HTTPException(status_code=400, detail=f"不支持的书目导出格式: {format}")
+        
+        # 执行导出
+        file_content, media_type, filename = export_service.export_library(format)
+        
+        # 对文件名进行URL编码，避免中文编码错误
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)